Bug: Unresolved dependencies error while building app

**Summary:**

Can't build the app, because of compiler unresolved dependencies errors, such as:

> Failed to resolve: multidex
> Failed to resolve: multidex-instrumentation

**Steps to reproduce:** 

1. Clone development branch
2. Open project in your IDE
3. Build project

**Expected behaviour:**

Complete build

**Observed behaviour:**

As I mentioned before, there are unresolved dependencies errors:

![image](https://user-images.githubusercontent.com/18665370/47484032-f7174400-d83a-11e8-9c3b-0e94987db872.png)

Running `./gradlew --debug check` logs:
```
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:getDependencies'.
Could not find multidex.jar 
```
For more details, check pastebin where I've posted more details from logs
 https://pastebin.com/XiXbYFAW

**Device and Android version:**

Mobile device doesn't matter here, so I'll just say that I'm using Android Studio as an IDE

**Screenshots:**

![image](https://user-images.githubusercontent.com/18665370/47484124-42c9ed80-d83b-11e8-9905-b47ad510db7f.png)

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -116,6 +116,9 @@ dependencies {
     //mifos passcode
     implementation "com.mifos.mobile:mifos-passcode:$mifosPasscodeVersion"
 
+    //multidex
+    implementation "com.android.support:multidex:$rootProject.multiDexVersion"
+
     // Unit tests dependencies
     testImplementation "junit:junit:$rootProject.jUnitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.mockitoVersion"

--- a/build.gradle
+++ b/build.gradle
@@ -19,8 +19,8 @@ buildscript {
 
 allprojects {
     repositories {
-        jcenter()
         google()
+        jcenter()
         maven { url "https://www.jitpack.io" }
         maven { url 'https://maven.fabric.io/public' }
     }
@@ -59,5 +59,6 @@ ext {
     sweeterrorVersion = '1.0.0'
     mifosPasscodeVersion = '0.3.0'
     cropviewVersion = '1.1.7'
+    multiDexVersion = '1.0.3'
 
 }

